core/spec/models/spree/calculator/default_tax_spec.rb

```diff
@@ -9,6 +10,11 @@
   let(:included_in_price) { false }
   subject(:calculator) { Spree::Calculator::DefaultTax.new(calculable: rate ) }
 
+  describe ".description" do
```

Comment:
@gevann I second this thought. In the individual calculator specs, we only want to see something like `it_behaves_like "a calculator with a description"`. The `describe` block can go into the shared example.